lib/events.js

```diff
@@ -456,10 +456,8 @@ function init_app(c) {
     // This requires the mouse wheel jquery plugin.
```

Comment:
Besides my other comment, if you have to make any other changes, maybe this comment should also be removed.